SOF-165-contactgegevens

Darkshadenl opened the pull request:
Description
Created a contact screen displaying the contact info of the cliënt. Portrait only.

Review focus
If the design is OK, and the code is maintainable. 

Checklist
+ I have run yarn format
+ I have run yarn eslint and fixed any issues
+ This PR does not add any errors to the console

A build will automatically be run by GitHub actions when any changes are made on this PR. This must complete successfully before merging.

Darkshadenl committed:
Contactgegevens scherm af

Darkshadenl committed:
Small changes

Darkshadenl committed:
Merge branch 'develop' into feature/SOF-165-contactgegevens

GewoonJaap commented on src/screens/HelpScreen.tsx:
```diff
@@ -1,13 +1,152 @@
 import MasterContainer from '../components/generic/MasterContainer';
+import FONTS from '../assets/fonts';
+import COLORS from '../assets/colors';
+import AS from '../assets/accessibilityStrings';
```
Misschien hier `ACCESSIBILITY_STRINGS` van maken, is iets duidelijker dan AS

Darkshadenl replied:
Behalve dan het feit dat ik iedere kans aangrijp om 'Ass' te schrijven of iets in de buurt daarvan, vind ik ACCESSIBILITY_STRINGS net te lang. ACC_STRS ook goed?

GewoonJaap replied:
Ja hoor, `ACC_STRS` lijkt me ook goed 👍

GewoonJaap reviewed (changes requested):
Code ziet er goed uit, had 1 klein dingetje. Zal m later vandaag/morgen nog even op IOS testen om te kijken of die het daar ook goed doet

GewoonJaap reviewed (changes requested):
De e-mail knop lijkt niet te werken op IOS. Als ik er op klik krijg ik een unhandled promise error.
![Schermafbeelding 2022-04-21 om 08 59 36](https://user-images.githubusercontent.com/33700526/164393185-649cf87e-02cb-41b5-bcc2-86519157064e.png)
Klopt de styling op ios? Omdat het zo ver naar links staat?

Darkshadenl commented:
> De e-mail knop lijkt niet te werken op IOS. Als ik er op klik krijg ik een unhandled promise error. ![Schermafbeelding 2022-04-21 om 08 59 36](https://user-images.githubusercontent.com/33700526/164393185-649cf87e-02cb-41b5-bcc2-86519157064e.png) Klopt de styling op ios? Omdat het zo ver naar links staat?

![image](https://user-images.githubusercontent.com/26344464/164451264-c9d4b9a3-0911-49a9-ab13-d4b5f68a8f30.png)

Blijkbaar werkt de link wel gewoon maar niet in de IOS simulator. 
Het design kan ik niet checken op IOS.

Darkshadenl committed:
Feedback processed

- Changed AS to ACC_STRS
- Email button color darker
- Email button works for IOS (probably)
- Commented out some information.
- Added call button

Darkshadenl committed:
formatted

GewoonJaap commented on src/screens/HelpScreen.tsx:
```diff
@@ -15,6 +15,18 @@ const createText = () => {
+  let phoneNumber = '';
+
+  if (Platform.OS === 'android') {
+    phoneNumber = 'tel:${1234567890}';
```
Waarvoor is t ${}? Want normaal kan je dat in javascript gebruiken om makkelijk variabele in een string te doen, alleen dan moet je well \` \` gebruiken ipv ' '. Evt dat het telefoon nummer nog als const ergens kan?

Darkshadenl replied:
whoops foutje.

Ik heb de phoneNumber in de `${}` gezet aangezien dit iets variabels wordt per helpscherm. 
Op dit moment is hier nog geen data voor, dus heb ik een TODO toegevoegd en phoneNumber gefilled met ACC._STRS.contactPhone.

GewoonJaap commented on src/screens/HelpScreen.tsx:
```diff
@@ -119,34 +145,34 @@ const styles = StyleSheet.create({
+    text: `${ACC_STRS.contactPostalCode} ${ACC_STRS.contactCity}`,
     style: styles.contactText,
   },
+  // {
```
Zou dit gewoon helemaal weg laten als het toch uitgecomment is

Darkshadenl replied:
Mijn gedachte is dat het best wel handig is om nog even te behouden. 
Dit scherm  gaat (mocht het verder uitegewerkt worden aangezien het meer een nice to have is) later nog door veel development heen. Deze data (testdata?) staat al klaar. Dan kan je lekker snel aan de slag gaan. 
Commenten kan geen kwaad.

Darkshadenl committed:
Little mistake fixed
